### PR TITLE
Update content and screenshots

### PR DESCRIPTION
**Pull request by anandcpm**
Re-structure DevOptics workshop to decouple from Core, and organize the labs by value prop. Updated content and screenshots to match new UX for DevOptics and GitHub

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,49 @@
 # CloudBees DevOptics Workshop
-This workshop will show how DevOptics allows organizations to measure, analyze and manage DevOps; and how DevOptics provides a comprehensive view of the status of your CD platform and software value streams, correlated from data collected across software delivery pipelines. 
+This workshop will provide a basic understanding of how to create Value Streams, monitor the Platform at scale and continually measure DevOps performance leveraging features of CloudBees DevOptics.
```
Comment (kmadel) [inline README.md]: Perhaps make CloudBees DevOptics a link to docs?

File: Setup.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+* https://github.com/cloudbees-days/helloapp-nodejs
+
+<p><img src="img/setup/setup_fork_repo.png" width=700/>
+
+## Back to the Jenkins Setup
+
+You should see the following **Create a new Pipeline** screen for your Team:
+<p><img src="img/setup/setup-success.png" width=700/>
+
+You may proceed to the next set of [**labs**](./README.md#workshop-labs) - once you have successfully completed the setup.
```
Comment (kmadel) [inline Setup.md]: Maybe just - You may proceed to the [**labs**] - since this isn't really a lab.

File: insights.md [hunk 3]
```diff
@@ -10,10 +11,9 @@
 * Optimal time for maintenance/upgrades
 
 ## Take a Tour of DevOptics Run Insights
-1. Go to https://devoptics.cloudbees.com in your browser and login with the email **beedemo.guest@gmail.com** and the password provided by your instructor <p><img src="img/insights/devoptics_login.png" width=800/>
-2. Click on the graph icon in the left menu to switch to the Run Insights view <p><img src="img/insights/insights_view.png" width=800/>
-3. Next click on the **?** left menu item and then select **Tour Run Insights** from the pop-up menu <p><img src="img/insights/tour_run_insights.png" width=800/>
-4. Complete the tour and then continue to the next exercise <p><img src="img/insights/start_tour.png" width=800/>
+Run insights provide you with a real-time and historical view of job and pipeline activity. You can connect any number of masters for an aggregate view across the organization. Masters can be a heterogeneous mix of CloudBees Core and Jenkins. Metrics are viewable over an adjustable time period.
```
Comment (kmadel) [inline insights.md]: CloudBees Core Masters and OSS Jenkins masters

File: insights.md [hunk 4]
```diff
@@ -24,10 +24,11 @@ Run Insights lets you filter the shown data per master or group of masters to un
 * It stores the filter setting in your local storage, to persist the filter when you return.
 
 
-1. The initial view will show you insights for all Jenkins Masters that are connected to the ***cloudbees-days*** account. To filter by your Team Master search for your Team Master name in the **Filter by Connected Masters...** search input and select your Team Master from the list <p><img src="img/insights/insights_filter_by_master_input.png" width=800/>
+1. The initial view will show you insights for all Jenkins Masters that are connected to the ***cloudbees-dwjw*** organization. To filter by your Team Master search for your Team Master name in the **Filter by Connected Masters...** search input and select your Team Master from the list <p><img src="img/insights/insights_filter_by_master_input.png" width=850/>
```
Comment (kmadel) [inline insights.md]: "connected to the ***cloudbees-dwjw*** organization" - You may want to make this generic as different orgs may be used for different workshops - like we did for CB Days.

File: value-streams-as-code.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+}
+```
+5. Click the **Save changes** button <p><img src="img/code/save_json.png" width=800/>
+6. You will have a new Value Stream <p><img src="img/code/microservice_vs.png" width=800/>.
+
+### The **helloapp-nodejs** Application
+
+The **helloapp-nodejs** application is a simple NodeJS app that serves a simple index page with a message. 
+
+If you navigate to **helloapp-nodejs** in your Jenkins master, you'll notice that there are two stages, a `Web Tests` stage used to test the NodeJS app using a selenium sandbox which runs on the `development` branch and a `Build and Push Image` stage that runs on `master`. <p><img src="img/code/hellonodejs_dev_branch.png" width=800/>
```
Comment (kmadel) [inline value-streams-as-code.md]: "using a selenium sandbox" it actually uses Testcafe - so maybe replace with: 

using a [TestCafe](https://devexpress.github.io/testcafe/) container

File: value-streams-as-code.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+5. Now that you have fixed your **helloapp-nodejs** application it is time to merge to the **master** branch and deploy. Create a [Pull Request](https://help.github.com/en/articles/creating-a-pull-request) between the **development** branch and **master** branch of your forked **helloapp-nodejs** repository. 
+6. Changed the **base repository** to the **master** branch of your forked **helloapp-nodejs** repository (not the **cloudbees-dwjw** repository), add a comment and then click the **Create pull request** button <p><img src="img/code/ui_dev_to_master_pr.png" width=600/>
+7. A job will be created for the pull request and once it has completed successfully your pull request will show that **All checks have passed**. Go ahead and click the **Merge pull request** button and then click the **Confirm merge** button but DO NOT delete the **development** branch <p><img src="img/code/ui_merge_pr.png" width=600/>
+8. In your DevOptics Value Stream you should see the **UI-1001** ticket move from the **UI Dev** gate to the **UI Master** gate. What is your deployment frequency? <p><img src="img/code/final_microservice_vs.png" width=800/>
```
Comment (kmadel) [inline value-streams-as-code.md]: Just curious why you spell out "deployment frequency" here but refer to MTTR above? Maybe at least put DF in parentheses.

File: value-streams.md [hunk 7]
```diff
@@ -7,205 +8,93 @@ A CloudBees DevOptics Value Stream models a complex continuous delivery process
...
+In the first set exercises we will setup a few sample Jenkins pipelines by [creating a GitHub Org Folder Pipeline project](#create-a-github-organization-folder-pipeline-project), get an overview of the [basic fundamentals of CloudBees DevOptics](#login-to-cloudbees-devoptics), and learn how to [configure a Value Stream in DevOptics mapped to a sample Jenkins pipeline](#value-stream-visual-editor). But first we have to [add some credentials to our Team Master](#add-github-credentials-to-your-team-master) - a **Username with password** credential for the GitHub Org Folder Pipeline project.
+
+## Add GitHub Credentials to Your Team Master
+
+We must switch to the Jenkins classic UI to complete the steps in this exercise.
+
+1. Click the ***Go to classic*** button at the top of common section of Blue Ocean’s navigation bar. <p><img src="img/intro/go_to_classic.png" width=800/>
```
Comment (kmadel) [inline value-streams.md:17]: Click the ***Go to classic*** button at the top of common section - reads a bit weird

Perhaps:
Click the ***Go to classic*** button at the top of the common section

File: value-streams.md [hunk 8]
```diff
@@ -7,205 +8,93 @@ A CloudBees DevOptics Value Stream models a complex continuous delivery process
...
...r*](https://jenkins.io/doc/book/pipeline/multibranch/#organization-folders) and sometimes more specifically a *GitHub Organization* project (this type of project is also [available for Bitbucket](https://plugins.jenkins.io/cloudbees-bitbucket-branch-source) and there is [unofficial support for GitLab](https://github.com/Argelbargel/gitlab-branch-source-plugin)). The Jenkins *GitHub Organization* project will scan a GitHub Organization to discover the Organization’s repositories, automatically creating **managed** [*Multibranch Pipeline* jobs](https://jenkins.io/doc/book/pipeline/multibranch/#creating-a-multibranch-pipeline) for any repository with at least one branch containing a *project recognizer* - typically **Jenkinsfile**. We will use the GitHub Organization that you created in **[Setup - Create a GitHub Organization](./Setup.md#create-a-github-organization)**. A Jenkins *GitHub Organization* project will also utilize a GitHub Organization level ***webhook it creates*** to automatically manage Jenkins jobs - both individual branch jobs and Mutlibranch Pipeline jobs associcated to repositories - when a branch or a repository is deleted from or added to the GitHub Organization.
```
Comment (kmadel) [inline value-streams.md:34]: "and there is [unofficial support for GitLab](https://github.com/Argelbargel/gitlab-branch-source-plugin)"
There is official Jenkins OSS support now:
"and [for GitLab](https://github.com/jenkinsci/gitlab-branch-source-plugin/)"

File: value-streams.md [hunk 9]
```diff
@@ -7,205 +8,93 @@ A CloudBees DevOptics Value Stream models a complex continuous delivery process
...
+
+In order to complete the following exercise you should have [forked the following repositories](./Setup.md#fork-the-workshop-repository) into the Github Organization you created in **[Setup - Create a GitHub Organization](./Setup.md#create-a-github-organization)**:
+
+* https://github.com/cloudbees-days/helloapp-api
+* https://github.com/cloudbees-days/helloapp-nodejs  
+
+Once that repository is forked:
```
Comment (kmadel) [inline value-streams.md]: "Once that repository is" to "Once those repositories are"

File: value-streams.md [hunk 10]
```diff
@@ -7,205 +8,93 @@ A CloudBees DevOptics Value Stream models a complex continuous delivery process
...
+2. Click on **New Item** in the left navigation menu - make sure that you are in the folder with the same name as your team, and not at the root of your Team Master  <p><img src="img/intro/org_folder_bluesteel_folder.png" width=800/>
+3. Enter your GitHub Organization name as the **Item Name** 
+4. Select **GitHub Organization** as the item type
+5. Click **Ok** <p><img src="img/intro/org_folder_item.png" width=800/>
+6. Select the credentials you created above from the **Credentials** drop down 
+7. Make sure that the **Owner** field matches the name of your GitHub Organization name
+8. Click the **Save** button <p><img src="img/intro/org_folder_save.png" width=800/>
+9. Click on the bread-crumb link to go back to your **GitHub Organization** Jenkins Pipeline project folder
+10. You should now see the two forked **helloapp** repositories as Jenkins jobs! <p><img src="img/intro/org_scan_helloapp_repos.png" width=800/>
```
Comment (kmadel) [inline value-streams.md]: "as Jenkins jobs" to "as Jenkins Multibranch Pipeline jobs"

File: value-streams.md [hunk 11]
```diff
@@ -7,205 +8,93 @@ A CloudBees DevOptics Value Stream models a complex continuous delivery process
...
+11. Click on **Open Blue Ocean** in the left navigation menu - to open list of jobs displayed in *Blue Ocean* UI.<p><img src="img/intro/org_scan_helloapp_blueocean_ui.png" width=850/>
```
Comment (kmadel) [inline value-streams.md]: "to open list of jobs" to "to view the Pipeline jobs"

File: value-streams.md [hunk 12]
```diff
@@ -7,205 +8,93 @@ A CloudBees DevOptics Value Stream models a complex continuous delivery process
...
+12. View the pipeline jobs in *Blue Ocean* UI 
+Note: We may switch between *Classic* and *Blue Ocean* UIs during the workshop, as needed - *Classic* UI would be utilized for detailed logs and administration purposes while *Blue Ocean* UI would be primarily used for viewing and running Jenkins pipelines. <p><img src="img/intro/setup_blue_ocean_ui_helloapp.png" width=800/>
+13. Let's review your first sample pipeline - **{your GitHub organization}/helloapp-api**. Verify if all branches for  *helloapp-api* are in a successful state, after the pipeline runs are completed. <p><img src="img/intro/setup_blue_ocean_helloapi_success.png" width=800/>
+16. Now let's verify the build status of our second sample pipeline - **{your GitHub organization}/helloapp-nodejs**. The *development* branch should be in a failed state, whereas the *master* branch should be successful state. <p><img src="img/intro/setup_blue_ocean_hellonodejs.png" width=800/>
```
Comment (kmadel) [inline value-streams.md]: "whereas the *master* branch should be successful state" to "whereas the *master* branch should have completed successfully"

File: value-streams.md [hunk 13]
```diff
@@ -7,205 +8,93 @@ A CloudBees DevOptics Value Stream models a complex continuous delivery process
...
+17. This completes the setup for DevOptics workshop! In short time, you created your very own CloudBees Core Team (Jenkins) Master, onboarded a few sample multi-branch pipelines and did basic walk through to get more comfortable navigating the CloudBees Core / Jenkins Master UI!
```
Comment (kmadel) [inline value-streams.md]: "the setup for DevOptics workshop" to "the setup for the DevOptics workshop"

Comment (kmadel) [reply value-streams.md]: "and did basic walk through" to "and did a basic walk through"

**Review by kmadel — CHANGES_REQUESTED** [review]
Looks pretty good, I just have a few suggestions.

File: value-streams-as-code.md (hunk 5 above)
Comment (anandcpm) [reply value-streams-as-code.md]: Updated.

File: insights.md (hunk 3 above)
Comment (anandcpm) [reply insights.md]: Updated - CloudBees Core Masters and Open Source Jenkins masters

File: Setup.md (hunk 2 above)
Comment (anandcpm) [reply Setup.md]: Make sense. Noted.

File: README.md (hunk 1 above)
Comment (anandcpm) [reply README.md]: Sure, updated Readme. Thanks.

File: value-streams-as-code.md (hunk 6 above)
Comment (anandcpm) [reply value-streams-as-code.md]: Not sure, that question was left over from baseline cb days content. 
Removed that, and replaced with a new screenshot focusing on value stream metrics.

File: value-streams.md (hunk 9 above)
Comment (anandcpm) [reply value-streams.md]: Good catch! Updated.

File: value-streams.md (hunk 10 above)
Comment (anandcpm) [reply value-streams.md]: Noted.

File: value-streams.md (hunk 11 above)
Comment (anandcpm) [reply value-streams.md]: Updated.

File: value-streams.md (hunk 12 above)
Comment (anandcpm) [reply value-streams.md]: Noted.

File: value-streams.md (hunk 13 above)
Comment (anandcpm) [reply value-streams.md]: Updated.

File: insights.md (hunk 4 above)
Comment (anandcpm) [reply insights.md]: Thanks - updated as follows: "...connected to your assigned organization configured in CloudBees DevOptics"

**Comment by anandcpm** [comment]
@kmadel - done, updated per suggestions. 
Please review and merge PR if all ok. Thanks!

**Review by kmadel — APPROVED** [review]

**Pull request merged**